app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/editTitleOrDescription/EditTitleOrDescriptionRoot.kt

```diff
@@ -0,0 +1,38 @@
+package com.davidtomas.taskyapp.features.agenda.presentation.editTitleOrDescription
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import com.davidtomas.taskyapp.features.agenda.presentation._common.navigation.AgendaRoutes
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+fun EditTitleOrDescriptionRoot(
+    navController: NavHostController,
+    editTitleOrDescriptionViewModel: EditTitleOrDescriptionViewModel = koinViewModel(),
```

Comment:
What if the requirements change and there's another type of text you can edit for an agenda item, e.g. a meeting link? Will you then call this `EditTitleOrDescriptionOrMeetingLinkViewModel`? 😄 

What I'm trying to say: The whole point is being able to edit some sort of text, so why not just call this `EditTextViewModel`? That makes it independent of the requirements.